public/index.html

```diff
@@ -5,8 +5,8 @@
     <!-- <link rel="icon" href="%PUBLIC_URL%/favicon1.ico" /> -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta
-      name="Ashutosh's Portfolio"
```

Comment:
Any Reason behind removing this?

Reply:
No reason @ashutosh1919 i will put it back as it is.